ValheimPlus/GameClasses/Attack.cs

```diff
@@ -59,4 +60,57 @@ private static void Postfix(ref Attack __instance, ref float __result)
             }
         }
     }
+
+    /// <summary>
+    /// Alter projectile velocity and accuracy without affecting damage
+    /// </summary>
+    [HarmonyPatch(typeof(Attack), "ProjectileAttackTriggered")]
+    public static class Attack_ProjectileAttackTriggered_Patch
+    {
+        private static void Prefix(ref Attack __instance)
+        {
+            if (Configuration.Current.ProjectileFired.IsEnabled)
+            {
+                if (__instance.m_character.IsPlayer())
+                {
+                    if (Configuration.Current.ProjectileFired.playerProjectileEnableScaling)
+                    {
+                        Player player = (Player)__instance.m_character;
+                        Skills.Skill skill = player.m_skills.GetSkill(__instance.m_weapon.m_shared.m_skillType);
+                        float maxLevelPercentage = skill.m_level * 0.01f;
+
+                        __instance.m_projectileVelMin = Mathf.Lerp(Configuration.Current.ProjectileFired.playerProjectileVelMinCharge, Configuration.Current.ProjectileFired.playerProjectileVelScaledMin, maxLevelPercentage);
+                        __instance.m_projectileVel = Mathf.Lerp(Configuration.Current.ProjectileFired.playerProjectileVelMaxCharge, Configuration.Current.ProjectileFired.playerProjectileVelScaledMax, maxLevelPercentage);
+
+                        __instance.m_projectileAccuracyMin = Mathf.Lerp(Configuration.Current.ProjectileFired.playerProjectileVarMinCharge, Configuration.Current.ProjectileFired.playerProjectileVarScaledMin, maxLevelPercentage);
+                        __instance.m_projectileAccuracy = Mathf.Lerp(Configuration.Current.ProjectileFired.playerProjectileVarMaxCharge, Configuration.Current.ProjectileFired.playerProjectileVarScaledMax, maxLevelPercentage);
+                    }
+                    else
+                    {
+                        __instance.m_projectileVelMin = Configuration.Current.ProjectileFired.playerProjectileVelMinCharge;
+                        __instance.m_projectileVel = Configuration.Current.ProjectileFired.playerProjectileVelMaxCharge;
+
+                        __instance.m_projectileAccuracyMin = Configuration.Current.ProjectileFired.playerProjectileVarMinCharge;
+                        __instance.m_projectileAccuracy = Configuration.Current.ProjectileFired.playerProjectileVarMaxCharge;
+                    }
+                }
+                else
+                {
+                    __instance.m_projectileVelMin = Configuration.Current.ProjectileFired.projectileVelMinCharge;
+                    __instance.m_projectileVel = Configuration.Current.ProjectileFired.projectileVelMaxCharge;
+
+                    __instance.m_projectileAccuracyMin = Configuration.Current.ProjectileFired.projectileVarMinCharge;
+                    __instance.m_projectileAccuracy = Configuration.Current.ProjectileFired.projectileVarMaxCharge;
+                }
+
+                float maxValue = 1e+6f;
```

Comment:
IMO, since this is a constant you should move it to the class scope.

Reply:
Yeah I probably when overboard with the change. @nxPublic told me to remove the constant and make them config so I thought it is bad practice to have class const. I rather have it as class const as well.